fetch_telemetry.py

```diff
@@ -31,114 +32,148 @@
 BIN_DIR = "/data/mta4/Script/SOH"
 HTML_DIR = "/data/mta4/www/CSH"
 HOUSE_KEEPING = f"{BIN_DIR}/house_keeping"
+
 #
-#--- append path to a private folder
+#--- Append path to a private folder
 #
 sys.path.append(BIN_DIR)
-import check_msid_status    as cms
+import check_msid_status as cms
 
 #
 #--- Defining Globals
+#
 BLOB_SECTIONS = ['ccdm', 'eps', 'load', 'main', 'mech', 'pcad', 'prop', 'sc_config', 'smode', 'snap', 'thermal']
-FETCH_SECONDS = 120
+FETCH_SECONDS = 30
```

Comment:
Changed the timing to 30 seconds as I've found that rarely the end of comm blob fetches will be too large with a two minute interval.